python/calico/felix/config.py

```diff
@@ -479,6 +484,12 @@ def _validate_cfg(self, final=True):
                 self.parameters["DefaultEndpointToHostAction"]
             )
 
+        if len(self.LOG_PREFIX) < 1:
+            raise ConfigException(
```

Comment:
I'd err of the side of not raising an exception here.  If the user _wants_ to set an empty log prefix, why not let them?

I'd also prevent defaulting over raising since, if you set this config in etcd, _every_ felix will validate it here and crash if the config is bad.